<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="3" parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="3" parallel="tests" name="testcase_3(failed)">
    <classes>
      <class name="TestScenario1.Testscript_36">
        <methods>
          <include name="Testurllogin"/>
          <include name="Testscenario"/>
        </methods>
      </class> <!-- TestScenario1.Testscript_36 -->
    </classes>
  </test> <!-- testcase_3(failed) -->
  <test thread-count="3" parallel="tests" name="testcase_2(failed)">
    <classes>
      <class name="TestScenario1.Testscript_37">
        <methods>
          <include name="Testurllogin"/>
          <include name="Testscenario"/>
        </methods>
      </class> <!-- TestScenario1.Testscript_37 -->
    </classes>
  </test> <!-- testcase_2(failed) -->
  <test thread-count="3" parallel="tests" name="testcase_1(failed)">
    <classes>
      <class name="TestScenario1.Testscript_38">
        <methods>
          <include name="Testurllogin"/>
          <include name="Testscenario"/>
        </methods>
      </class> <!-- TestScenario1.Testscript_38 -->
    </classes>
  </test> <!-- testcase_1(failed) -->
</suite> <!-- Failed suite [Suite] -->
